src/JuMP.jl

```diff
@@ -23,8 +23,6 @@ using Compat
 
 import Compat: UTF8String, view
 
-@compat import Base.show
-
```

Comment:
Probably. I broke 0.4 with this, need to fix up.

On Jul 29, 2016 09:04, "Jarrett Revels" <notifications@github.com> wrote:

> In src/JuMP.jl
> <https://github.com/JuliaOpt/JuMP.jl/pull/810#discussion_r72806821>:
>
> > @@ -23,8 +23,6 @@ using Compat
> >
> >  import Compat: UTF8String, view
> >
> > -@compat import Base.show
> > -
>
> I thought this was still needed for the writemime patch? See here
> <https://github.com/JuliaLang/Compat.jl/pull/219>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaOpt/JuMP.jl/pull/810/files/7e57d0c55f6b5089a3d70c6728258c9f90b32f18#r72806821>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABp0M4PaN-YYXClTQT0pYUVdDrYloohKks5qahZmgaJpZM4JYOi6>
> .
>